### PR TITLE
DDF-3766 Removed defaults label and resized sources dropdown

### PR DESCRIPTION
**Pull request by idperez**
#### What does this PR do?
Removes default label in settings.
Sources dropdown similarly aligned with other dropdowns. 

#### Who is reviewing it? 
@lcrosenbu 
@alexaabrd 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
@andrewkfiedler
@bdeining
#### What are the relevant tickets?
[DDF-3766](https://codice.atlassian.net/browse/ddf-3766)
#### Screenshots (if appropriate)
Before
screen shot 2018-05-02 at 9 18 32 am
screen shot 2018-05-02 at 9 17 13 am
After
screen shot 2018-05-02 at 8 56 33 am
screen shot 2018-05-02 at 8 56 44 am

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less [hunk 1]
```diff
@@ -78,17 +78,17 @@
   .dropdown-text {
     padding: 0px 10px;
     width: ~'calc(100% - @{minimumButtonSize})';
-    min-height: @minimumButtonSize;
+    height: @minimumButtonSize;
```
Comment (jefchng) [inline ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less]: When multiple sources are selected, they are separated by ` | ` and will expand to the next line if needed. I think this change will remove the ability for the list of selected sources to continue on the next line.

Comment (jefchng) [inline ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less]: Selected sources are separated by ` | ` and if needed, the list of sources would continue to the next line. I think this change limits the dropdown to a single line thus hiding part of the list.

**Comment by bdeining** [comment]
@idperez is it possible to post some screenshots?

**Comment by andrewkfiedler** [comment]
screen shot 2018-04-30 at 4 57 42 pm

I'm a little torn on this one.  Maybe we should be splitting defaults into a separate area entirely.  It's a bit deceptive to simply remove the label.  The `number of search results` is an option, while everything else is a default.

File: ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less (hunk 1 above)
Comment (idperez) [reply ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less]: screen shot 2018-05-02 at 8 59 03 am
This is showing with the min-height property set. Using height sizes it properly. I just checked and you were right about how the height property hides part of the selected list of sources.

**Comment by vinamartin** [comment]
❗️ Do we still need this @andrewkfiedler / @lcrosenbu ?
The contributor hasn't updated this PR in 4 months, is it still relevant?

**Comment by vinamartin** [comment]
Contributor is inactive and this change doesn't appear to be needed.
Let's reopen this if it still has value. Closing.

**Pull request closed without merging**